themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -80,73 +80,6 @@ <h4 class="dg-promos__heading">Join a Community</h4>
 {{ end }}
 {{ $services := ($.Scratch.Get "services") }}
```

Comment:
Looks like there's some services code starting at line 70.

Reply:
oh good call... need to look above the code as well to see if its related